src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/AEMExtensionTests.cs

```diff
@@ -70,5 +70,11 @@ public void TestAEMExtensionAdvancedLinux()
         {
             ComputeTestController.NewInstance.RunPsTest("Test-AEMExtensionAdvancedLinux");
         }
+
+        [Fact]
```

Comment:
Need to add properties to the Fact attribute: Category.AcceptanceType, Category.CheckIn, or the test will not run as part of check-in